source/braille.py

```diff
@@ -28,6 +28,7 @@
 from collections import namedtuple
 import re
 import scriptHandler
+import eventHandler

```

Comment:
Not too happy with another circular dependency, but I know we discussed this before and all solutions we came up with were hacky.